source/src/cip/cipconnectionmanager.c

```diff
@@ -207,7 +207,7 @@ EipStatus ConnectionManagerInit(EipUint16 unique_connection_id) {
   InitializeConnectionManagerData();
 
   CipClass *connection_manager = CreateCipClass(
-    g_kCipConnectionManagerClassCode,   /* class ID */
+    kCipConnectionManagerClassId,   /* class ID */
```

Comment:
same here, official name Class Code

Reply:
Here the constant keeps the "ClassCode" prefix but loses the "g_" prefix.